### PR TITLE
[1.19] images/kubekins-e2e: Update variants.yaml (add 1.19, drop 1.15)

### PR DESCRIPTION
**Pull request by cpanato**
Release cut issue: https://github.com/kubernetes/sig-release/issues/1136

Pending branch cut:
/hold

/sig release
/area release-eng

/assign @tpepper @justaugustus @cblecker 
cc: @kubernetes/release-engineering @kubernetes/sig-release-admins

### CODE REVIEW

**Comment by tpepper** [comment]
Probably lgtm, but depends slightly on what was happening in https://github.com/kubernetes/test-infra/pull/18169 with the markers?

**Comment by justaugustus** [comment]
@tpepper -- I'll rebase my PR later.

--

1.19 branch is live.
/hold cancel

File: images/kubekins-e2e/variants.yaml [hunk 1]
```diff
@@ -30,9 +36,3 @@ variants:
     K8S_RELEASE: stable-1.16
     BAZEL_VERSION: 2.2.0
     OLD_BAZEL_VERSION: 0.23.2
-  '1.15':
```
Comment (BenTheElder) [inline images/kubekins-e2e/variants.yaml:33]: are we running no more 1.15 CI?

Comment (BenTheElder) [reply images/kubekins-e2e/variants.yaml:33]: more specifically: this should come after 1.15 CI is shut down. is it?

Comment (tpepper) [reply images/kubekins-e2e/variants.yaml:33]: Is running https://testgrid.k8s.io/sig-release-1.15-blocking

Should not be.  Got missed a few months ago now.

Comment (justaugustus) [reply images/kubekins-e2e/variants.yaml:33]: @BenTheElder -- yep, it should be shut down. 1.15 jobs have already been removed: https://github.com/kubernetes/test-infra/pull/17867

Comment (spiffxp) [reply images/kubekins-e2e/variants.yaml:33]: Mostly shut down.  The dashboard @tpepper linked shows the generated jobs are still running (eg: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/generated/generated.yaml#L1860-L1887) though I'm just now hilariously noticing that many of the jobs aren't using release-branch variants of kubekins.  Whoops.

I'm fine merging this despite that.  The jobs that use kubekins 1.15 are just going to be stuck on their current image until removed (https://github.com/kubernetes/test-infra/search?q=kubekins+1.15&unscoped_q=kubekins+1.15&type=Code)

Comment (justaugustus) [reply images/kubekins-e2e/variants.yaml:33]: @spiffxp -- that's the PR to follow. Once this merges and the new kubekins-e2e images get built, I'll be able to rotate the jobs.

**Pull request merged**